.circleci/config.yml

```diff
@@ -1943,6 +1944,7 @@ jobs:
             cd /; GIT_SSH_COMMAND="ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no" \
               git clone  --recurse-submodule ssh://git@github.com/swirlds/swirlds-platform.git
             cd /swirlds-platform; git submodule update --init --recursive --checkout
+            cd /swirlds-platform; git checkout 83d6374; git submodule update
```

Comment:
I have created a  branch `services-regression` in platform and platform apps  with that tag. We can use it here  intead of `83d6374`